climada/hazard/test/test_base_xarray.py

```diff
@@ -551,6 +551,7 @@ def test_2D_time(self):
         np.testing.assert_array_equal(
             hazard.date, [val.toordinal() for val in time.flat]
         )
+        np.testing.assert_array_equal(hazard.event_name, ["1999-01-01", "1999-02-01", "2000-01-01", "2000-02-01"])
```

Comment:
black would format like this
```suggestion
        np.testing.assert_array_equal(
            hazard.event_name, ["1999-01-01", "1999-02-01", "2000-01-01", "2000-02-01"]
        )
```